apps/site/assets/ts/stop/models/displayTimeConfig.ts

```diff
@@ -77,6 +77,9 @@ const infoToDisplayTime = (
   ];
 
   const [departureInfo1, departureInfo2] = getNextTwoTimes(departureInfos);
+  const isCr =
+    departureInfo1?.routeMode === "commuter_rail" ||
+    departureInfo2?.routeMode === "commuter_rail";
```

Comment:
This could probably be a helper function in `departure-info.ts`.
Something like `isCommuterRail(d1) || isCommuterRail(d2)`